### ISSUE TITLE
Cant reference stylus module from `node_module` dir

### ISSUE THREAD

**Issue by bencripps** [jbwyme/babel-plugin-stylus-compiler#3]
In a file, built through webpack we are requiring a `node_module` that is our core style library.

``` css
@require '~mozu-adminui-core-style'
```

However, when I run this through the babel-plugin, it tries to require the following  

`~mozu-adminui-core-style.styl` rather than finding the associated file from the package.json

Is this something you'd like to support? Seems like it would be helpful? Let me know your thoughts.

**Comment by bencripps** [comment]
I've dug in a bit, and the execution is failing on this line:

``` js
  return stylus(stylusContent)
    .include(dirname(path))
    .include(resolve('./node_modules'))
    .use(autoprefixer())
    .render()
    .replace(/\n/g,' ');
```

specifically, on the `render` call; I see that we're including the `./node_modules` directory -- which is where my package is located, but it looks like stylus doesn't understand that the `~` represents a node_module rather than a explicit file location.

This actually makes sense, since the `~` syntax is taken directly from `stylus-loader` functionality.

> > stylus-loader can also take advantage of webpack's resolve options. With the default options it'll find files in web_modules as well as node_modules, make sure to prefix any lookup in node_modules with ~.

Here's a link to their documentation; https://github.com/shama/stylus-loader

I'd like to get this functionality into your plugin, as I think it would be nice to have feature parity with style loader. Let me know your thoughts; I can create a MR to try and add this feature if you like.

**Comment by jbwyme** [comment]
Yeah I actually ran into this myself since we use the stylus-loader in
other parts of the code base. I'm good with having parity on that if you
want to open up a PR.

**Comment by bencripps** [comment]
Ok sounds good, I'll fork and submit a MR.

**Comment by bencripps** [comment]
I've got a branch going, but when I run the tests (and I've reverted to the tip of master), the tests seem to fail). 

![image](https://cloud.githubusercontent.com/assets/6272507/18146043/4cec5b44-6f93-11e6-95e8-b18d26f99c83.png)

Any idea? Do they pass for you?

**Comment by bencripps** [comment]
false alarm -- I cleaned my `node_modules` directory and the tests now pass.

**Comment by bencripps** [comment]
@jbwyme I submitted a MR for this issue; let me know if you have any questions/concerns.

**Comment by jbwyme** [comment]
merged as of 1.4.0
